src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1092,6 +1092,47 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
         }
         #endregion
 
+        #region Configuration API
+        /// <inheritdoc/>
+        public override Task<IReadOnlyList<ConfigurationItem>> GetConfiguration(
+            string storeName,
+            IReadOnlyList<string> keys,
+            IReadOnlyDictionary<string, string> metadata = default,
+            CancellationToken token = default)
+        {
+            var request = new Autogenerated.GetConfigurationRequest()
+            {
+                StoreName = storeName
+            };
+            request.Keys.AddRange(keys);
+
+            if (metadata != null)
+            {
+                foreach (var kvp in metadata)
+                {
+                    request.Metadata.Add(kvp.Key, kvp.Value);
```

Comment:
Is request.metadata initialized by default? or no?

Reply:
It appears to be. I have a test where we set it and it's OK with this. Other requests involving a map do the same.